### PR TITLE
Mezmo

### PR DESCRIPTION
**Pull request by lstott**
First version of the Mezmo integration.

### What does this PR do?

Creates the initial (V1) Mezmo integration

### Motivation

To list the Mezmo integration.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: amixr/mezmo/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Agent Check: Mezmo Telemetry Pipeline
+
+## Overview
+
+Mezmo Telemetry Pipeline empowers teams to ingest, transform, and route telemetry data in real-time. You can ingest telemetry data from various sources via our open platform, apply transformation and processing logic, then route to Datadog.
```
Comment (emarsha94) [inline amixr/mezmo/README.md:5]: The overview section should include the following:

1. What does Mezmo do in 1-2 sentences?
2. What will customers get from this integration?
3. What types of data does this send in and the value of the data?

File: amixr/mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json
Comment (emarsha94) [inline amixr/mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json:1]: Please include a dashboard with your integration

File: amixr/mezmo/images/IMAGES_README.md
Comment (emarsha94) [inline amixr/mezmo/images/IMAGES_README.md:1]: This can be deleted

File: amixr/mezmo/manifest.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "1.0.0",
```
Comment (emarsha94) [inline amixr/mezmo/manifest.json:2]:
```suggestion
  "manifest_version": "2.0.0",
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Hi team! Thanks for submitting your PR. It looks like there is a duplication of all files in this PR. Can you remove all the duplicates or submit a PR with the just the files needed? Thanks!

**Comment by cswatt** [comment]
Created ticket for docs team to review readme.

File: mezmo/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Agent Check: Mezmo Telemetry Pipeline
+
+## Overview
+
+Mezmo Telemetry Pipeline empowers teams to ingest, transform, and route telemetry data in real-time. You can ingest telemetry data from various sources via our open platform, apply transformation and processing logic, then route to Datadog.
```
Comment (emarsha94) [inline mezmo/README.md:5]: The overview section should include the following:

What does Mezmo do in 1-2 sentences?
What will customers get from this integration?
What types of data does this send in and the value of the data?

File: mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json
Comment (emarsha94) [inline mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json:1]: Please include a dashboard

File: mezmo/images/IMAGES_README.md
Comment (emarsha94) [inline mezmo/images/IMAGES_README.md:1]: Please remove this file

File: mezmo/manifest.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "1.0.0",
+  "app_uuid": "3f460698-08ba-4693-bc1b-d5ecef6c6701",
+  "app_id": "mezmo-telemetry-pipeline",
+  "display_on_public_website": false,
```
Comment (emarsha94) [inline mezmo/manifest.json:5]:
```suggestion
  "display_on_public_website": true,
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Comment by aliciascott** [comment]
Hi @lstott thank you for this contribution! Can you review and make the suggested changes from @emarsha94 and then we can move forward with the editorial review. Thanks!

File: mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json
Comment (lstott) [reply mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json:1]: Hi, do you have any sample dashboards I can base this off of? Not a DD user so any help is appreciated.

File: mezmo/assets/logos/Mezmo_Logo_Charged-Yellow-Grounded-Green_RGB_Square.svg
[the remark this answers is not in the document]
Comment (lstott) [reply mezmo/assets/logos/Mezmo_Logo_Charged-Yellow-Grounded-Green_RGB_Square.svg:1]: Is it due to the size? if you download the logo you can see it's not a black square.

File: amixr/mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json
Comment (anawishnoff) [reply amixr/mezmo/assets/dashboards/mezmo_telemetry_pipeline_overview.json:1]: Hi @lstott, you can view our dashboarding best practices here which was examples and guidelines: https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices

File: mezmo/assets/logos/Mezmo_Logo_Charged-Yellow-Grounded-Green_RGB_Square.svg
[the remark this answers is not in the document]
Comment (anawishnoff) [reply mezmo/assets/logos/Mezmo_Logo_Charged-Yellow-Grounded-Green_RGB_Square.svg:1]: This looks fine now

**Comment by anawishnoff** [comment]
Hi @lstott, once the comments from @emarsha94 are addressed, we can do another round of reviews. Please feel free to ask any questions you may have on the requested changes. Thanks!

**Comment by lstott** [comment]
Hi Ana,

Since we are a Pipeline company (meaning we help transport and transform
data) and we do not have any unique data of our own, I'm struggling to come
up with a unique Mezmo Datadog dashboard. Is this a hard requirement or can
there be an exception made given the nature of the Mezmo pipeline?

**Comment by anawishnoff** [comment]
Hi @lstott, since you are sending metrics into Datadog as a part of this integration, we require an out of the box dashboard. It can be a simple dashboard, but should display the metrics that you're sending in, organized in a way that would make sense for customers.

The dashboard should also have your logo on it, and provide a description wtih links to any relevant documentation. Let me know if you have remaining concerns here. Thanks!

**Comment by lstott** [comment]
The metrics can be anything that our mutual customers decide to send in. I
don't have a list of metrics that will be sent every time that I could use
in an OOB dashboard. Does that make sense?

**Comment by anawishnoff** [comment]
Hi @lstott, as discussed over email and the recent call, this integration will need to send in specific metrics/logs/other data about the health of the Mezmo observability pipelines/other parts of your platform. An out of the box dashboard is also required to display all of that data in one place. 

Please let us know if you have any questions, thank you!

**Comment by emarsha94** [comment]
@lstott As discussed via email, we will go ahead and close this out until the integration has been completed. Feel free to re-open once you integration is completed and ready for review!

**Pull request closed without merging**